### ISSUE TITLE
Labels of RL and cot sft data

### ISSUE THREAD

**Issue by EchoDreamer** [ReasoningTransfer/Transferability-of-LLM-Reasoning#1]
Hi, very insightful work! I’d like to ask about the labels of RL data and cot SFT data in Section 2.2 — Are the training labels for RL from the ground-truth annotations of the originally collected data? And for the SFT data, it is distilled from the Qwen3-32B model. I’m wondering if there’s a possibility that the labels from the distilled data might not be entirely accurate, which could lead to poor transfer performance on mathematical and other reasoning benchmarks. Also, did you test the accuracy of the originally collected training data using the Qwen3-32B model? Furthermore, I observed in Table 1 that especially on the hallucination evaluation benchmark HaluEval for non-reasoning tasks, the SFT performance is very poor. Could it be that this is due to the distilled SFT data containing more hallucinated knowledge/reasoning, which affects the results? All in all, I am wondering if the experimental results in the paper might be due to the poor quality and hallucinations in the distilled SFT data rather than poor transfer performance of SFT.
